### PR TITLE
General Improvement suggestions

### LINKED ISSUE #69: Swifter framework removal & Cocoadebug framework replacement with Pulse
- Swifter framework should be deleted from the project because of lack of support and less usage
- Pulse framework should be integrated instead of cocoadebug.

### PR DESCRIPTION
**Pull request by egesucu**
Cocoapods suggest 3 warnings when installing/updating the pods.

1. [!] Automatically assigning platform `iOS` with version `13.0` on target `ios-boilerplate` because no platform was specified. Please specify a platform for this target in your Podfile. See `https://guides.cocoapods.org/syntax/podfile.html#platform`.
2. [!] The `ios-boilerplateUITests [Development]` target overrides the `ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES` build setting defined in `Pods/Target Support Files/Pods-ios-boilerplate-ios-boilerplateUITests/Pods-ios-boilerplate-ios-boilerplateUITests.development.xcconfig'. This can lead to problems with the CocoaPods installation
3. [!] The `ios-boilerplateUITests [Swifter]` target overrides the `ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES` build setting defined in `Pods/Target Support Files/Pods-ios-boilerplate-ios-boilerplateUITests/Pods-ios-boilerplate-ios-boilerplateUITests.swifter.xcconfig'. This can lead to problems with the CocoaPods installation

This PR will resolve all of them.

Also this PR will silence some warnings and recommendations when Xcode builds and runs the project.

### CODE REVIEW

File: ios-boilerplate.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1225,8 +1226,8 @@
 			isa = XCRemoteSwiftPackageReference;
 			repositoryURL = "https://github.com/httpswift/swifter";
 			requirement = {
-				kind = exactVersion;
-				version = 1.5.0;
+				branch = stable;
```
Comment (bahaulug-tech) [inline ios-boilerplate.xcodeproj/project.pbxproj]: Why did you change it from exactversion to stable ?

Comment (egesucu) [reply ios-boilerplate.xcodeproj/project.pbxproj]: Stable has never changes than 1.5.0. 1.5.0 is stopped being maintained in 2020 and stable branch stopped being maintained in 2021.

**Review by bahaulug-tech — APPROVED** [review]

**Review by cemal-yilmaz-adesso — APPROVED** [review]
